src/agent/rustjail/src/mount.rs

```diff
@@ -139,7 +139,7 @@ pub fn init_rootfs(
     for m in &spec.mounts {
         let (mut flags, data) = parse_mount(&m);
         if !m.destination.starts_with("/") || m.destination.contains("..") {
-            return Err(ErrorKind::Nix(nix::Error::Sys(Errno::EINVAL)).into());
+            return Err(anyhow!(nix::Error::Sys(Errno::EINVAL)));
```

Comment:
Curious what is the difference between `nix::Error::Sys(Errno::EINVAL))` and `anyhow!(nix::Error::from_errno(Errno::EINVAL)` (used above). From reading the doc, I see no real difference.

If there is no real difference, consider making the code consistent between the two locations.